mne/decoding/tests/test_transformer.py

```diff
@@ -229,3 +229,49 @@ def test_unsupervised_spatial_filter():
     usf = UnsupervisedSpatialFilter(PCA(4), average=True)
     usf.fit_transform(X)
     assert_raises(ValueError, UnsupervisedSpatialFilter, PCA(4), 2)
+
+
+def test_temporal_filterer():
+    """Test methods of TemporalFilter."""
+    X = np.random.rand(10, 5, 2)
+
+    # Add tests for different combinations of l_freq and h_freq
+    filt = TemporalFilter(l_freq=5, h_freq=15, sfreq=100,
+                          filter_length='auto', l_trans_bandwidth=None,
+                          h_trans_bandwidth=None)
+    with warnings.catch_warnings(record=True):  # stop freq attenuation warning
```

Comment:
yes I would just make a signal/set of param such that there s not warning, rather than catching it.